be/src/exec/workgroup/work_group.h

```diff
@@ -96,21 +103,29 @@ class WorkGroupManager {
 public:
     // add a new workgroup to WorkGroupManger
     void add_workgroup(const WorkGroupPtr& wg);
+
     // remove already-existing workgroup from WorkGroupManager
     void remove_workgroup(int wg_id);
-    // get next workgroup for computation
-    WorkGroupPtr pick_next_wg_for_cpu();
-    // get next workgroup for io
-    WorkGroupPtr pick_next_wg_for_io();
 
-    WorkGroupQueue& get_cpu_queue();
+    WorkGroupPtr get_workgroup(int wg_id);
 
+    // get next workgroup for io
+    WorkGroupPtr pick_next_wg_for_io();
     WorkGroupQueue& get_io_queue();
 
+    void log_cpu() {
+        for (const auto& [_, wg] : _workgroups) {
+            LOG(WARNING) << "[WorkGroup] "
+                         << "[name=" << wg->name() << "] "
+                         << "[cpu_limit=" << wg->get_cpu_limit() << "] "
+                         << "[vruntime_ms=" << wg->get_vruntime_ns() / 1000'000.0L << "] "
+                         << "[real_runtime_ms=" << wg->get_real_runtime_ns() / 1000'000.0L << "] ";
+        }
+    }
+
 private:
     std::mutex _mutex;
     std::unordered_map<int, WorkGroupPtr> _workgroups;
-    CpuWorkGroupQueue _wg_cpu_queue;
```

Comment:
Why remove the `CpuWorkGroupQueue` and keep the `IoWorkGroupQueue`?

Reply:
In this PR, we use `DriverQueueWithWorkGroup` instead of `CpuWorkGroupQueue`, so `CpuWorkGroupQueue()` is removed. However, I'm not sure whether `IoWorkGroupQueue` is needed.